pandas/core/resample.py

```diff
@@ -749,6 +749,24 @@ def size(self):
             result = pd.Series([], index=result.index, dtype='int64')
         return result
 
+    def quantile(self, q=0.5, **kwargs):
+        """
+        Return value at the given quantile.
+
+        Parameters
+        ----------
+        q : float or array-like, default 0.5 (50% quantile)
+
+        See Also
+        --------
+        Series.quantile
+        DataFrame.quantile
+        DataFrameGroupBy.quantile
+
+        .. versionadded:: 0.24.0
```

Comment:
the tag goes right below the summary line (the first line of the doc-string, with a blank line before and after)